net/dns/manager.go

```diff
@@ -274,7 +284,13 @@ func (m *Manager) EnqueuePacket(bs []byte, proto ipproto.Proto, from, to netaddr
 //
 // TODO(tom): Rip out once all platforms use netstack.
 func (m *Manager) NextPacket() ([]byte, error) {
-	resp := <-m.responses
+	var resp response
+	select {
+	case <-m.ctx.Done():
+		return nil, ErrClosed
```

Comment:
or ctx.Error()

Reply:
This needs to match in `wgengine.pollResolver`, so ill leave it as `net.ErrClosed`